ApollosProject/apollos-apps
# feat: DailyHabitsExperienceCard Component

## ✏️ Description
We are introducing a new card type that is **specific** to Daily Habits. This card will be rendered exclusively on the Daily Habits tab for the time being, so we're introducing this component with the name of: DailyHabitsExperienceCard.

For reference, you can see the [Figma Component](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=4401%3A68140) as well as the [Daily Habit Screen](https://www.figma.com/file/YHJLj8pdFxWG9npF2YmB3r/UI-Kit-2.0?node-id=4296%3A61742) for different use cases.

## 🔬 To Test
1. Set your `PROJECT_VARIANT` to `storybook`
2. `yarn && yarn start` then `yarn ios`
3. Check out the story

## 📸 Screenshots
| Light | Dark |
| --- | --- |
| ![CleanShot 2022-11-04 at 14 05 25](https://user-images.githubusercontent.com/45076058/200052367-df170ee6-005b-433d-8735-69c0cb45fa56.png) | ![CleanShot 2022-11-04 at 14 05 35](https://user-images.githubusercontent.com/45076058/200052391-638c8a3b-bec4-460e-92f9-e2fa4eddc8d8.png) |

## 📝 Checklist
